jmxtrans/pom.xml

```diff
@@ -316,8 +316,9 @@
 								<script>if [ $1 = 1 ]; then
                   USER_ID=`id -u %{package.user} 2&gt;/dev/null`
                     if [ -z "$USER_ID" ]; then
+									    groupadd ${package.user}
```

Comment:
Should you add a check to only create the group if it already exists? Or use the `-f` option?